diffraction/fft_smooth.py

```diff
@@ -0,0 +1,70 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2021 Scipp contributors (https://github.com/scipp)
+# flake8: noqa: E501
+"""
+This script implements a few functions in order to perform a simple data reduction of POWGEN data
+using scipp.
+"""
+
+import numpy as np
+import scippneutron as scn
+from scipy.fft import irfft
+from scipy.fft import rfft
+
+
+def fft_smooth(dataset, n, order):
```

Comment:
Specify dimension to smooth in as an argument.
Then, there is no need to check for `'tof'` or `'spectrum'` below.